src/common/logger.ts

```diff
@@ -1,6 +1,17 @@
 import * as bunyan from 'bunyan';
 import config = require('./config');
 
+if (process.env.LOG_LEVEL) {
+  try {
+    // Validate if LOG_LEVEL has valid bunyan logging level
+    bunyan.resolveLevel(process.env.LOG_LEVEL);
+  } catch {
+    delete process.env.LOG_LEVEL;
+  } finally {
+    config.LOGGING.level = process.env.LOG_LEVEL || config.LOGGING.level;
```

Comment:
can't this be moved to after the `bunyan.resolveLevel` call in the try clause? so we don't need finally at all.
```js
try {
    // Validate if LOG_LEVEL has valid bunyan logging level
    bunyan.resolveLevel(process.env.LOG_LEVEL);
    config.LOGGING.level = process.env.LOG_LEVEL;
  } catch {
    delete process.env.LOG_LEVEL;
  }
```

also - not sure why we need/want to delete the env var